index.bs

```diff
@@ -3220,11 +3203,9 @@ This attestation statement format is used with FIDO U2F authenticators using the
 
 : Signing procedure
 :: If the credential public key of the given credential is not of algorithm -7 ("ES256"), stop and return an error.
-    Otherwise, let |authenticatorData| denote the [=authenticator data for the attestation=], 
-    and let |clientDataHash| denote the [=hash of the serialized client data=].
-
-    If |clientDataHash| is 256 bits long, set |tbsHash| to this value. Otherwise set |tbsHash| to the SHA-256 hash of
-    |clientDataHash|.
+    Otherwise, let |authenticatorData| denote the [=authenticator data for the attestation=],
+    and let |tbsHash| denote the [=hash of the serialized client data=]. (Since SHA-256 is used to hash the
+    serialized client data, |tbsHash| will be 32 bytes long.)
```

Comment:
If you do rename `|tbsHash|` to `|clientDataHash|` as suggested in the next comment, it's probably good to do that here too.